src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/NotifyAttentionNeededLicencesService.kt

```diff
@@ -31,7 +32,10 @@ class NotifyAttentionNeededLicencesService(
       return
     }
 
-    val prisoners = prisonerSearchApiClient.searchPrisonersByNomisIds(attentionNeededLicences.map { it.nomsId!! })
+    val batchedNomsIds = attentionNeededLicences.map { it.nomsId!! }.chunked(Companion.BATCH_SIZE)
```

Comment:
no need to change but you don't need to refer to this via `Companion.`

Reply:
will change it in next PR.